test/parallel/test-fs-open.js

```diff
@@ -82,7 +82,7 @@ for (const extra of [[], ['r'], ['r', 0], ['r', 0, 'bad callback']]) {
     }
   );
   assert.throws(
-    () => fs.openSync(i, 'r', common.mustNotCall()),
+    () => fs.openSync(i, 'r'),
```

Comment:
`common.mustNotCall()` creates a function that throws if invoked. So having it there is a way of testing that `openSync()` does not invoke a callback. Removing it makes the test less effective.

Reply:
I understand. close it.